src/main/kotlin/no/nav/familie/ef/sak/journalføring/JournalføringService.kt

```diff
@@ -238,11 +239,12 @@ class JournalføringService(
         taskRepository.save(BehandlingsstatistikkTask.opprettMottattTask(behandlingId = behandlingId, oppgaveId = oppgaveId))
     }
 
-    private fun opprettSaksbehandlingsoppgave(behandling: Behandling, navIdent: String?): Long {
+    private fun opprettSaksbehandlingsoppgave(behandling: Behandling, navIdent: String?, mappeId: Long?): Long {
```

Comment:
Hvis denne settes til default null så trenger man ikke å sende med null på de andre stedene som egentlige ikke trenger den
```suggestion
    private fun opprettSaksbehandlingsoppgave(behandling: Behandling, navIdent: String?, mappeId: Long? = null): Long {
```

Reply:
Kan også fjerne den fra alle tester.. ?

Reply:
Vi diskuterte litt frem og tilbake om vi ville ha default null eller ikke. Var ikke helt sikker på hva som er "lurt" og mest idiotsikkert for fremtiden.

Reply:
Av og til er det fint å tvinge fram et valg, men vet ikke om dette er det beste eksempelet på et slikt behov.